roles/openshift_gcp/tasks/node_cloud_config.yml

```diff
@@ -6,6 +6,7 @@
   ini_file: dest=/etc/origin/cloudprovider/gce.conf section=Global option={{ item.key }} value={{ item.value }} state=present create=yes
   with_items:
     - { key: 'project-id', value: '{{ openshift_gcp_project }}' }
+    - { key: 'network-project-id', value: '{{ openshift_gcp_network_project }}' }
```

Comment:
This should have `openshift_gcp_network_project | default(openshift_gcp_project)`